### DISCUSSION TITLE
Downloading Sky Surveys

### DISCUSSION THREAD

**Discussion by TheTurntableSlave** [Stellarium/stellarium#2036] [General]
Hi
Sorry if this has been asked before, I couldn't see a result when using Search.

Is it possible to store Sky Surveys offline for quicker loading into the program? I only use one or two Surveys but that delay when streaming the data is something I'd love to reduce.

Thanks

**Comment by gzotti** [comment]
Please see chapter 10 of the User Guide. Most HiPS surveys are provided by professionals, and it may be difficult or even illegal to download and host them locally. If you manage to get yours, the process is described in section 10.2.

**Reply by TheTurntableSlave** [reply]
I have to admit... I don't even know where the user guide was! I looked for something similar before asking the question. All I saw was a wiki (without the answer) and some documentation over at Sourceforge (also without the answer). Now you mentioned it, I see it's on the top right of the main page. Thank you.

I've found the offline repository based on that document; I only need it because my remote location has awful internet so it'd be good to have that site "pre-loaded" as it were. Thanks again for your help

**Reply by gzotti** [reply]
Ah yes, the prehistoric sites...

We want to allow software archaeologists to analyze the project history, therefore this is still available. Really, the program has evolved since 2015!

@alex-w it seems I cannot edit that SF wiki, or maybe only sub-pages. We should have a big fat warning on the start page that this is heavily outdated and just shows the state of 2017, with a link to GH. And we should probably really weed out and ultimately close the old wiki.

**Reply by Quark-Coder** [reply]
I've got a local HiPS survey, but in Chapter 10.2 there are no instructions on how to set up local HiPS surveys — only for TOAST.

**Reply by 10110111** [reply]
@Quark-Coder local surveys are the same as remote ones, except you use the `file://` protocol instead of `http://` or `https://`.

**Comment by TheTurntableSlave** [comment]
Long time since I made this thread!  Is this how the setup is supposed to look? Does Stellarium look at the folder, the .tar file or should I move all of the individual files currently in the "11" folder into the parent folder?

![Screenshot 2024-09-08 153431](https://github.com/user-attachments/assets/cdd608d6-972c-4a5f-ba03-353bc73bed02)

**Reply by gzotti** [reply]
The respective section in the User Guide meanwhile is number 10.4. You can read:

>The images are stored in subdirectories that ...

There is no mention of TAR files, so yes, you must unpack them. 

Why should you move level 11 files to the parent folder?

The Guide has
> `toast_survey_host=T:/`

From which follows by visual inspection and comparison: No space after C. A drive name under DOS/Windows has always, since the 1980s, looked like C:

Oh, and I do hope you have downloaded the other levels as well into their respective subdirectories!
